runtime/runtime/src/verifier.rs

```diff
@@ -429,7 +438,7 @@ pub fn validate_action(
         Action::CreateAccount(_) => Ok(()),
         Action::DeployContract(a) => validate_deploy_contract_action(limit_config, a),
         Action::DeployGlobalContract(a) => validate_deploy_global_contract_action(limit_config, a),
-        Action::UseGlobalContract(_) => validate_use_global_contract_action(),
+        Action::UseGlobalContract(a) => validate_use_global_contract_action(a),
```

Comment:
Should we add some comment in this function that would suggest to the implementer of new actions they may need to validate `account_id` here?